managing_providers/_topics/editing_a_containers_provider.md

```diff
@@ -70,6 +70,8 @@ Edit information about a provider such as the name, hostname, IP address or port
 
 7.  Under **Endpoints** in the **Alerts** tab, configure the following for Prometheus alerting from the cluster.
 
+    **Note:** Prometheus alerts are not supported on Red Hat OpenShift Container Platform version 4.
+
```

Comment:
The only version of OCP we support is 4, so this comment ends up creating a null situation.  I think it's better to just drop this entire section.

Note that we've dropped the entire OpenShift Alerts pages as well as the Monitor top-level menu for this reason.  Can you see if there are other docs to drop as well related to that?